### DISCUSSION TITLE
Influence of the "skill level" on the likelihood calculation

### DISCUSSION THREAD

**Discussion by c-bless** [cutaway-security/IACS_STAR_Methodology#5] [General]
While doing some tests with the IACS Risk calculator I was struggling with the calculation of the Likelihood Factor.

One part for calculating the **Likelihood Factor** is the **Threat Actor Factor**. For this the **Skill level** is used among others. There you have options from 0 - 9 (e.g. 1 = Limited IT, network and no OT skills, 9 = Security penetration skills and knowledge of OT). 

Selecting a higher value results in a Higher **Threat Actor Factor** and therefore in a higher **Likelihood Factor**. Meaning it is more likely that a professional threat actor cause damage. 

Seeing it from another point of view, one could argue that it might be more likely that someone abuse a vulnerability when the required skill level is lower. With less required skills the amount of possible threat actors might be much higher. Since **Threat Actor Factor** is used for calculation of Likelihood you could think about ordering the options for **Skill level** in the other way round.

**Comment by cutaway** [comment]
@c-bless I understand you reasoning but I have to disagree. The overall Threat Actor Factors are a combination of the skill, opportunity, motivation, and size scores. I think your reasoning is balanced by the opportunity and motivation scores. Any of the groups can have an impact on the process if their efforts provide access. Just because less skilled people don't know the process does not mean the other groups won't make a mistake and have the same effect. 

I do think the methodology needs an update when it comes to the Threat Actor Factors. I don't think the external consultant should be selecting these. The consultant should be working with the stake holders to define these selections and use them for each of the calculations for an assessment. I don't think the selections need to change, just how we describe the ratings and use during an assessment for a specific scope.

That said, please point others to this conversation so we can get their input on your recommendation.

Another thought, the good thing about this calculator is that the users can easily reverse the selection. Just because the statement is there for the score doesn't mean that statement automatically applies. You can select what ever number you feel best represents the rating. In these cases a note needs to be made in documentation to identify differences from the norm.
